### ISSUE TITLE
Can you tell if a scheduled event is running or not?

### ISSUE THREAD

**Issue by michaelmaston** [parnic/node-screenlogic#56]
Thanks so much for this project.  Super useful.

In the ScreenLogic app when you look at the scheduled events (at least on the iOS version), it highlights jobs in green when they are running.  Any idea if this running state is queried from the controller or is the app just being smart and looking at the current time and the state of the circuit and assuming the schedule is the reason for it being active?  The reason I ask is I notice the onboard clock can get off over time and the time window drift because of it so assuming both controller and my code are on the same time isn't a great assumption.   I'm assuming that the controller has a way to query this info for the app but I'm not seeing it in the code/doc.

Related question, is there a way to get the current time on the controller to determine if it is reasonably in synch?

Thanks!!

**Comment by parnic** [comment]
I just loaded up the iOS ScreenLogic app on my test iPod and wow, it's a lot nicer than the Android version!

I'm not finding anything right away in the Android source code about whether a schedule is active or not, but that doesn't mean it doesn't exist. If it does exist, I don't know where it is right now. If you find it, I'd love to accept a PR!

I was able to find a way to get and set the current ScreenLogic system's time, though, so I've added that to the library.
